specification/objects/barrier.md

```diff
@@ -2,39 +2,36 @@
-		- **Privileges Required**:
-		
-			*Type Specific Properties* <br />
-			**neededPrivileges** (one): The privileges that are needed relative to the type of barrier being overcome. (See [Privilege Levels](../values/privilege-level.md))
-		- **Encryption**: Data or access to communication is protected through a mechansism that converts infomration into code or cipher.
- - **Obfuscation**: The attacker must overcome a protection mechansim in place that deliberately creates confusing or difficult to determine values or locations for resources or data.
-	- **ASLR**: Some form of Address space layout randomization (ASLR) is in use that must be defeated to recognize impacts
-	- **Binary Diversity**: The relationship of the source code and the binary is non-predictable
- - **State**:  Unique conditions exist either in process timing, implementation, architecural environment or particular configurations of a given system or systems within its influence. 
-    - **Race Condition**:  The exploit scenario includes requiring an attacker to take advantage of a race condition. Race Condition has multiple [Race Condition Values](../values/race-condition.md) that can be assigned. 
-    - **Specialized Condition**:  The exploit scenario requires specific, non-default configuration settings within the vulnerable software. For example, the use of a non-standard port for a networked service like ssh.
-    - **Environmental Conditon**:  The exploit scenario requires an environmental condition external to the vulnerable software that is not necessarily related to the vulnerable software itself. A congested network would be an example of an environmental condition.
-    - **Precondition Required**:  Information about the target is necessary in order to exploit the vulnerability on a specific target. For example, the hostname of the device may be necessary in order to exploit the vulnerability on that device.
-  - **Boundary Protections**:  The attacker must be able to move through explicit boundaries that otherwise would prevent successful exploitation
-  
-	*Type Specific Properties* <br />
-  **relatesToContext** (one): The context to which the boudary is related (See [Context](../values/context.md)).
-    - **Sandbox**:  The product is deployed within a sandbox that must be broken out from to recognize impacts
-    - **Container**:  The product is deployed within a type of container that must be broken out from to recognize impacts.
+## Properties
+
+- **id** (one): A globally unique identifier for the impact that distinguishes it from other impacts related to the same vulnerability.
+- **hasBarrierType** (one): Identifies the kind of barrier. Based on the barrier's type, [additional properties](#additional-properties) may be required.
```

Comment:
Should add something similar to below, there is no connector to the new Types .md created.
(See [Barrier Types](../values/barrier-type.md))